docs/source/howto/installation.rst

```diff
@@ -547,20 +547,30 @@ See the :doc:`../reference/_changelog` for a list of breaking changes.
 
 .. _how-to:installation:backup:
 
-Backing up your installation
+Backing up your data
 ============================
 
-A full backup of an AiiDA instance and AiiDA managed data requires a backup of:
+General information
+-----------------
 
-* the AiiDA configuration folder, which is named ``.aiida``.
-  The location of the folder is shown in the output of ``verdi status``.
-  This folder contains, among other things, the ``config.json`` configuration file and log files.
+The most convenient way to back up the data of a single AiiDA profile is to use
 
-* the data stored for each profile.
-  Where the data is stored, depends on the storage backend used by each profile.
+.. code:: bash
 
-The panels below provide instructions for storage backends provided by ``aiida-core``.
-To determine what storage backend a profile uses, call ``verdi profile show``.
+    $ verdi --profile <profile_name> storage backup /path/to/destination
+
+This command is safe to use when AiiDA is running, it automatically manages a subfolder structure of previous backups, and new backups are done in an efficient way (using ``rsync`` hard-link functionality to the previous backup).
+The command backs up everything that's needed to restore the profile later:
+
+* the AiiDA configuration file ``.aiida/config.json``, from which other profiles are removed (see ``verdi status`` for exact location);
+* all the data of the backed up profile (which depends on the storage backend).
+
+The specific procedure and whether it is even implemented depends on the storage backend.
+
+Storage backend specific information
+-----------------
+
```

Comment:
The docs build is failing because these underlines are too short. Please correct them and check that the build succeeds, or fix the problem.

Reply:
whoops! thanks, fixed.